src/components/TimeRangePicker/TimeRangePicker.js

```diff
@@ -0,0 +1,80 @@
+import PropTypes from "prop-types";
+import { useEffect, useState } from "react";
+import { FiClock } from "react-icons/fi";
+import { MdOutlineKeyboardArrowDown } from "react-icons/md";
+import clsx from "clsx";
+import { TimeRangePickerBody } from "./TimeRangePickerBody";
+import "./index.css";
+import { defaultRange } from "./rangeOptions";
+import { convertRangeValue, createDisplayValue, storage } from "./helpers";
+
+export const TimeRangePicker = ({ onChange }) => {
```

Comment:
The value must come from outside. If need be you can use `valueTo` and `valueFrom` instead of `value`
```suggestion
export const TimeRangePicker = ({ onChange, value }) => {
```